app/controllers/Login.scala

```diff
@@ -26,6 +26,6 @@ class Login @Inject()(config: ApplicationConfig, override val controllerComponen
   val show = Action(
     Redirect(
       config.basGatewaySignInUrl,
-      Map("continue_url" -> Seq(controllers.routes.Dashboard.home().url), "origin" -> Seq("voa"))))
+      Map("continue_url" -> Seq(config.newDashboardUrl("home")), "origin" -> Seq("voa"))))
```

Comment:
At some point can we change to `config.dashboardUrl("home")` - if it's not too much work, maybe this PR?